wordpress-mobile/WordPress-iOS
# Change the loading indicator from a ghost view to an activity indicator

Closes #14080

This PR removes the ghost animation and replaces it with an activity indicator, to let the user know that their search results are loading.

### To test
I used FG pages to test this because there are many and it takes a while to load, even on a fast connection.

1. Install and launch the application.
2. Tap on "Log In".
3. Log in to the app using valid credentials.
4. Open "My site" tab.
5. Tap on "Menus".
6. Select or add new menu.
7. Tap on New item and list will open.
8. Tap on any option from the list (eg. Page,Link, Tag, Post,Product)
9. Observe the ghost view has been replaced with a spinner.
10. Try out a few searches that have no results ("Wow"). The spinner should disappear and a message for no results should appear.
11. Try a search with only a couple of results ("Show"). The spinner should show up below the cached results and eventually disappear when it doesn't find anything new.

### Screenshots
**iOS 13**
| Light mode | Dark mode |
| --- | --- |
| ![Simulator Screen Shot - iPhone 11 Pro - 2020-11-04 at 11 06 19](https://user-images.githubusercontent.com/1062444/98146772-6d6ec800-1e91-11eb-9421-c9ed2ab844a0.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-11-04 at 11 08 50](https://user-images.githubusercontent.com/1062444/98146802-72cc1280-1e91-11eb-87c5-112d929824e1.png) |

| iOS 12 |
| --- |
| <a href="https://user-images.githubusercontent.com/1062444/98146872-7f506b00-1e91-11eb-9c6a-d21c3c976dda.png"><img src="https://user-images.githubusercontent.com/1062444/98146872-7f506b00-1e91-11eb-9c6a-d21c3c976dda.png" width="350" /></a> |

PR submission checklist:
